test/e2e/config/fixture.yaml

```diff
@@ -32,3 +34,6 @@ intervals:
   default/wait-deployment: ["5m", "1s"]
   default/wait-namespace-deleted: ["20s", "1s"]
   ironic/wait-deployment: ["10m", "2s"]
+  default/wait-provisioning: ["20s", "1s"]
+  default/wait-provisioned: ["1m", "10ms"]
+  default/wait-deprovisioning: ["1m", "2s"]
```

Comment:
Although we are not testing this currently, shouldn't we also have `default/wait-available` declared here as we have in ironic.yaml . Whats the plan for running fixture by the way?

Reply:
Good catch, I've missed that one, Also noticed that ``default/wait-registration`` was missing. I have both ready for a commit if needed. What do you think @lentzi90 ?

Reply:
Good find! Yes this is needed or we get a timeout when running the fixture provider. We can use the same value as for `inspection/wait-available`.

> Whats the plan for running fixture by the way?

There is no plan currently. I try to make sure I run it manually (locally on my laptop) for every PR for now. The fixture tests are not important per se, but they give a good indication if the tests are flexible enough to be able to run on bare metal. I.e. if we can run fixture, then there is no dependency on libvirt or similar.
Maybe we should add a test that runs the fixture tests just for this reason, what do you think?

Reply:
Yeah thats a good point, I was suggesting mainly from the point since we are introducing changes in this file in every tests and it would be good to test them also, otherwise issues will just pile up, but if you are testing in your laptop already, there wouldn't be much issues. Anyways, its good to have it in CI incase if someone forgets to test it.

Reply:
Added an issue for it: https://github.com/metal3-io/baremetal-operator/issues/1406